### PR TITLE
New OS: QTECH

### PR DESCRIPTION
**Pull request by moisseev**
Add new OS QTECH.

QTECH company offers several lines of Ethernet switches designed specifically for Internet providers, data centers, industrial enterprises and the energy sector.

https://www.qtech.ru/en/catalog/

QSW-2500-24-AC

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Thank you for your contribution.  
Could you add hardware and version support? Need to do that in php code. Look at some other os for examples

File: tests/data/qtech.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
                     "sysObjectID": ".1.3.6.1.4.1.27514.1.1.2.7",
                     "sysDescr": "QSW-3500-24-AC Device, Compiled Feb 22 10:28:35 2011\n SoftWare Version QSW-3500-24-AC_6.0.220.41\n BootRom Version QSW-3500-24-AC_2.0.2\n HardWare Version R01\n Copyright (C) 2010 by QTECH LLC\n All rights reserved",
                     "sysContact": "<private>",
-                    "version": null,
-                    "hardware": null,
+                    "version": "QSW-3500-24-AC_6.0.220.41",
+                    "hardware": "R01",
```
Comment (Jellyfrog) [inline tests/data/qtech.json:29]: Isn't "QSW-3500-24-AC" the hardware?

Comment (moisseev) [reply tests/data/qtech.json:29]: It is reported by one of the switch models as a part of software version. Of course, it is possible to capture just the version number, but I have at hand only two models of switches and have no idea about version string patterns of others. So I would leave it as is for compatibility.

```
SNMPv2-MIB::sysDescr.0 = STRING: QSW-3500-24-AC Device, Compiled Feb 22 10:28:35 2011
 SoftWare Version QSW-3500-24-AC_6.0.220.41
 BootRom Version QSW-3500-24-AC_2.0.2
 HardWare Version R01
 Copyright (C) 2010 by QTECH LLC
 All rights reserved
```
```
SNMPv2-MIB::sysDescr.0 = STRING: QSW-3450-28T-AC Device, Compiled Dec 30 11:22:40 2014
 SoftWare Version 7.0.3.5(R0237.0034)
 BootRom Version 7.1.36
 HardWare Version R01
  Serial No.:00030661509010026177
 Copyright (C) 2014 by QTECH LLC
 All rights reserved
```

Comment (Jellyfrog) [reply tests/data/qtech.json:29]: seems a bit messy   
Either way, `QSW-3500-24-AC` and `QSW-3450-28T-AC` must be the hardware right?  
So you should change that.  

Also, could you add your other device to the tests? You can name the test file like `qtech_qsw3450`

Comment (moisseev) [reply tests/data/qtech.json]: ok, I just thought `hardware` is a `revision`, but it is a `platform` actually.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-64-release-changelog-may-2020/12290/1

**Pull request merged**